src/test/java/org/jenkinsci/plugins/workflow/test/steps/FakeTest.java

```diff
@@ -0,0 +1,54 @@
+package org.jenkinsci.plugins.workflow.test.steps;
+
+import java.util.logging.Logger;
+
+import org.jenkinsci.plugins.workflow.cps.CpsFlowDefinition;
+import org.jenkinsci.plugins.workflow.graph.FlowGraphWalker;
+import org.jenkinsci.plugins.workflow.graph.FlowNode;
+import org.jenkinsci.plugins.workflow.job.WorkflowJob;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.jvnet.hudson.test.BuildWatcher;
+import org.jvnet.hudson.test.RestartableJenkinsRule;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.notNullValue;
+
+public class FakeTest {
+
+    @ClassRule
+    public static BuildWatcher bw = new BuildWatcher();
+
+    @Rule
+    public RestartableJenkinsRule rj = new RestartableJenkinsRule();
+
+    private static final Logger LOGGER = Logger.getLogger(FakeTest.class.getName());
+
+    @Test
+    public void fakeStepExecutes() {
+        rj.then(j -> {
+            WorkflowJob p = j.createProject(WorkflowJob.class, "p");
+            p.setDefinition(new CpsFlowDefinition("fake 20", true));
+            var run = p.scheduleBuild2(0).waitForStart();
+            j.waitForMessage("fake", run);
```

Comment:
```suggestion
            j.waitForMessage("going to sleep for", run);
```
would be clearer and more robust